pkg/mapper/iac-providers/cft/mapper.go

```diff
@@ -0,0 +1,213 @@
+package cft
+
+import (
+	"errors"
+
+	"github.com/awslabs/goformation/v4/cloudformation/cloudfront"
+	"github.com/awslabs/goformation/v4/cloudformation/cloudtrail"
+
+	cf "github.com/awslabs/goformation/v4/cloudformation/cloudformation"
+	cnf "github.com/awslabs/goformation/v4/cloudformation/config"
+	"github.com/awslabs/goformation/v4/cloudformation/ecr"
+	"github.com/awslabs/goformation/v4/cloudformation/neptune"
+	"github.com/awslabs/goformation/v4/cloudformation/secretsmanager"
+	"github.com/awslabs/goformation/v4/cloudformation/workspaces"
+
+	"github.com/awslabs/goformation/v4/cloudformation/ec2"
+	"github.com/awslabs/goformation/v4/cloudformation/efs"
+	"github.com/awslabs/goformation/v4/cloudformation/elasticache"
+
+	"github.com/awslabs/goformation/v4/cloudformation/dax"
+	"github.com/awslabs/goformation/v4/cloudformation/dynamodb"
+	"github.com/awslabs/goformation/v4/cloudformation/rds"
+
+	"github.com/awslabs/goformation/v4/cloudformation/ecs"
+	"github.com/awslabs/goformation/v4/cloudformation/logs"
+
+	"github.com/accurics/terrascan/pkg/iac-providers/output"
+	"github.com/accurics/terrascan/pkg/mapper/core"
+	"github.com/accurics/terrascan/pkg/mapper/iac-providers/cft/config"
+	"github.com/accurics/terrascan/pkg/mapper/iac-providers/cft/store"
+	"github.com/accurics/terrascan/pkg/utils"
+	"github.com/awslabs/goformation/v4"
+	"github.com/awslabs/goformation/v4/cloudformation"
+	"github.com/awslabs/goformation/v4/cloudformation/amazonmq"
+	"github.com/awslabs/goformation/v4/cloudformation/apigateway"
+	"github.com/awslabs/goformation/v4/cloudformation/apigatewayv2"
+	"github.com/awslabs/goformation/v4/cloudformation/docdb"
+	"github.com/awslabs/goformation/v4/cloudformation/elasticloadbalancing"
+	"github.com/awslabs/goformation/v4/cloudformation/elasticloadbalancingv2"
+	"github.com/awslabs/goformation/v4/cloudformation/elasticsearch"
+	"github.com/awslabs/goformation/v4/cloudformation/guardduty"
+	"github.com/awslabs/goformation/v4/cloudformation/iam"
+	"github.com/awslabs/goformation/v4/cloudformation/kinesis"
+	"github.com/awslabs/goformation/v4/cloudformation/kinesisfirehose"
+	"github.com/awslabs/goformation/v4/cloudformation/kms"
+	"github.com/awslabs/goformation/v4/cloudformation/redshift"
+	"github.com/awslabs/goformation/v4/cloudformation/route53"
+)
+
+const errUnsupportedDoc = "unsupported document type"
+
+type cftMapper struct {
+}
+
+// Mapper returns an CFT mapper for given template schema
+func Mapper() core.Mapper {
+	return cftMapper{}
+}
+
+// Map transforms the provider specific template to terrascan native format.
+func (m cftMapper) Map(doc *utils.IacDocument, params ...map[string]interface{}) (output.AllResourceConfigs, error) {
+	allRC := make(map[string][]output.ResourceConfig)
+	t, err := extractTemplate(doc)
```

Comment:
any unmarshalling errors by the goformation library are also logged to console. eg:
`ERROR: json: cannot unmarshal string into Go struct field SecurityGroup_Ingress.Properties.SecurityGroupIngress.FromPort of type int`
we should see how these logs can be suppressed.